assets/src/application/ui/layout/entityList/EntityList/index.tsx

```diff
@@ -49,10 +39,18 @@ const EntityList = ({
 	}
 
 	const classes = classNames('ee-editor-entity-list', htmlClass);
-	return view === 'grid' ? (
-		<EntityGridView entities={filteredEntities} className={classes} {...otherProps} />
-	) : (
-		<EntityListView entities={filteredEntities} className={classes} {...otherProps} />
+	const renderView =
+		view === 'grid' ? (
+			<EntityGridView entities={filteredEntities} className={classes} {...otherProps} />
+		) : (
+			<EntityListView entities={filteredEntities} className={classes} {...otherProps} />
+		);
+
+	return (
+		<>
+			{renderView}
```

Comment:
Could be this way:
```tsx
	const Component = view === 'grid' ? EntityGridView : EntityListView;

	return (
		<>
			<Component entities={filteredEntities} className={classes} {...otherProps} />
			<EntityPagination {...paginationProps} showSizeChanger />
		</>
```

Reply:
Yes. great suggestion, thx.
Took that and also added a bit of context to that component name: https://github.com/eventespresso/event-espresso-core/pull/2431/commits/be670944bf3a14cd6807ba92a4e3b6f875f7a816